CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+# 0.5.0 - Apr. 29, 2025
```

Comment:
Q about ordering. Shouldn't this PR have been merged first into main, and then branch the release PR off of the first commit with this change log?

Reply:
Even with a release PR with upgraded binaries, you could still merge the change log into main first? On the other hand, it's nice for a release PR to contain the change log, that's probably more true.